nucliadb/nucliadb/writer/api/v1/upload.py

```diff
@@ -754,3 +755,12 @@ def maybe_b64decode(some_string: str) -> str:
     except ValueError:
         # not b64encoded
         return some_string
+
+
+def guess_content_type(filename: str) -> str:
+    default_ct = "application/octet-stream"
+    try:
+        guessed_ct, _ = mimetypes.guess_type(filename)
```

Comment:
When it can raise a `TypeError`?

Reply:
It happens if for some reason filename is None. It should not happen though and it should be catched by mypy if we have such a situation. So I will remove it 👍🏼